### PR TITLE
Improve farmer dashboard

### PR DESCRIPTION
**Pull request by djerfy**
## Request(s)

* Add harvesters count
* Add partials performance

## Screenshot(s)

image

### CODE REVIEW

File: src/app/explorer/farmer/farmer.component.html [hunk 1]
```diff
@@ -54,38 +54,54 @@ <h3 class="display-3" i18n>Farmer Details</h3>
...
+                      <span class="text-warning text-uppercase" i18n>Failed<br/>Partials</span>
                       <p class="display-3">{{ partialsFailed }}</p>
                     </div>
                   </div>
                 </div>
+                <div class="col-lg-2 justify-content-center">
+                  <div class="card card-lift shadow border-0">
+                    <div class="card-body py-3 text-center">
+                      <span class="text-info text-uppercase" i18n>Partials<br/>Performance</span>
+                      <p class="display-3" ngbTooltip="{{ 100 - ( partialsFailed * 100 / partialsSuccessful ) | number:'2.6' }}%">{{ 100 - ( partialsFailed * 100 / partialsSuccessful ) | number:'2.1-1' }}%</p>
```
Comment (william-gr) [inline src/app/explorer/farmer/farmer.component.html:93]: Wonder what would happen in typescript here if partialsSuccessful were 0? xD

Comment (djerfy) [reply src/app/explorer/farmer/farmer.component.html:93]: image
 🤕

Comment (william-gr) [reply src/app/explorer/farmer/farmer.component.html:93]: Meh, accurate enough :D

Comment (william-gr) [reply src/app/explorer/farmer/farmer.component.html:93]: I was too fast with the merge button, sorry.
You wouldn't need length, but it could work

**Pull request merged**